app/views/models/welcome.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for(:title, _("Hardware Models")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("info-circle", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Hardware Models') %></h1>
+  <p><%= _("Hardware Model describes the hardware model of your hosts, including CPU class, Vendor class and general information") %></p>
```

Comment:
1. The first fragment doesn't quite make sense, should this be: "Hardware models describe the hardware types of your hosts"?
1. Don't capitalise "Vendor".
1. "General information" is unclear in this context, perhaps call them "other notes".
1. Missing full stop.